<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="LT0021232" name="com.ril.feedzai.testScript.AM_LMW_TranDetails" tests="6" failures="1" timestamp="15 Jun 2018 06:18:53 GMT" time="20.477" errors="0">
  <testcase name="P2M_Tran_Data" time="16.654" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails"/>
  <testcase name="LMW_Instrument_details" time="0.442" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails"/>
  <testcase name="LMW_Customer_dt" time="1.320" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails"/>
  <testcase name="LMW_transaction_details" time="0.062" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails">
    <skipped/>
  </testcase> <!-- LMW_transaction_details -->
  <testcase name="LMW_transaction_details" time="0.089" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails">
    <failure type="java.lang.AssertionError" message="Source Institution ID not found for transaction in transaction details section">
      <![CDATA[java.lang.AssertionError: Source Institution ID not found for transaction in transaction details section
at org.testng.Assert.fail(Assert.java:96)
at com.ril.feedzai.testScript.AM_LMW_TranDetails.LMW_transaction_details(AM_LMW_TranDetails.java:109)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LMW_transaction_details -->
  <testcase name="LMW_transaction_details" time="1.910" classname="com.ril.feedzai.testScript.AM_LMW_TranDetails">
    <skipped/>
  </testcase> <!-- LMW_transaction_details -->
</testsuite> <!-- com.ril.feedzai.testScript.AM_LMW_TranDetails -->
